sdk/sqlmanagement/Azure.ResourceManager.Sql/api/Azure.ResourceManager.Sql.netstandard2.0.cs

```diff
@@ -9705,6 +10445,24 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         public static bool operator !=(Azure.ResourceManager.Sql.Models.PerformanceLevelUnit left, Azure.ResourceManager.Sql.Models.PerformanceLevelUnit right) { throw null; }
         public override string ToString() { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct PricingModel : System.IEquatable<Azure.ResourceManager.Sql.Models.PricingModel>
```

Comment:
Too generic

Reply:
Renamed to SqlManagedInstancePricingModel